filament/src/materials/dofDilate.mat

```diff
@@ -54,26 +54,32 @@ vec2 dilate(inout vec2 center, const vec2 tap) {
 }
 
 void postProcess(inout PostProcessInputs postProcess) {
-    // highp is not needed here, because we now the resolution is very low
-    vec2 uv = variable_vertex.xy;
-    vec2 size = vec2(textureSize(materialParams_tiles, 0));
-    ivec2 xy = ivec2(uv * size);
+    const float radius = 1.0;
+    // we need highp here just to maintain precision of 1/x
+    highp vec2 size = vec2(textureSize(materialParams_tiles, 0));
+    highp vec2 isize = 1.0 / size;
+
+    vec2 radii = float(radius) * isize;
 
-    vec2 center = tap(xy, ivec2(0, 0));
+    vec2 uv = variable_vertex.xy;
```

Comment:
Shouldn't this be highp?

Reply:
ah no, because the resolution of tiles is very small -- we assume highp is not needed.